src/view/utils.js

```diff
@@ -48,4 +48,11 @@ export function getVm(n) {
 	while (n.vm == null && n.parent)
 		n = n.parent;
 	return n.vm;
+}
+
+export function getVnode(el) {
+	el = el || emptyObj;
```

Comment:
when would `el` be falsey? is there a case where `evt.target` is falsey?

Reply:
Not currently, but I was wanting to continue the discussion about exposing this function as part of the API. We did briefly discuss that as potentially useful at the app level.

Also, I tend to lean a little defensive in the utility function design when the logical expectation of a null input would be a null output, especially if null output is already a possibility.

In this case, I could go either way -- null input is invalid and causes a fast-failure, or it's valid and causes a null output.